test/run_tests.py

```diff
@@ -211,6 +211,45 @@ def test_inference():
     assert_dir("model_output/llik")
     # os.removedirs("model_output")
 
+def test_inference1():
```

Comment:
let's rename this test to something more informative.

Reply:
We can rename it depending upon the  changes in config asks us to do.